test/libsyncengine/jobs/network/testnetworkjobs.cpp

```diff
@@ -46,18 +46,22 @@
 #include "libparms/db/parmsdb.h"
 #include "utility/jsonparserutility.h"
 #include "requests/parameterscache.h"
+#include "test_utility/temporarydirectory.h"
 
 using namespace CppUnit;
 
 namespace KDC {
 
 static const SyncPath localTestDirPath(std::wstring(L"" TEST_DIR) + L"/test_ci");
```

Comment:
```suggestion
static const SyncPath localTestDirPath(Utility::s2ws(TEST_DIR) + L"/test_ci");
```

Might be defined in a header if it is re-used extensively.